composeml/label_times.py

```diff
@@ -136,18 +136,71 @@ def bin(self, bins, labels=None, right=True, quantiles=False):
                     The range is extended by .1% on each side to include the minimum and maximum values.
                     If `quantiles` is `True`, defines the number of quantiles (e.g. 10 for deciles, 4 for quartiles, etc.)
-                * array : If `quantiles` is `False`, defines the bin edges allowing for non-uniform width. No extension is done.
-                    If `quantiles` is `True`, defines array of quantiles (e.g. [0, .25, .5, .75, 1.] for quartiles)
+                * bins (array) : Bin edges either user defined or quantile-based.
+                    If `quantiles` is `False`, defines the bin edges allowing for non-uniform width. No extension is done.
+                    If `quantiles` is `True`, defines the bin edges usings an array of quantiles (e.g. [0, .25, .5, .75, 1.] for quartiles)
 
             labels (array) : Specifies the labels for the returned bins. Must be the same length as the resulting bins.
             right (bool) : Indicates whether bins includes the rightmost edge or not. Does not apply to quantile-based bins.
             quantiles (bool) : Determines whether to use a quantile-based discretization function.
 
         Returns:
             LabelTimes : Instance of labels.
+
+        Examples
+        --------
+        .. _equal-widths:
+
+        Using bins of `equal-widths`_:
+
+        >>> labels.bin(2).head()
+                                        my_labeling_function
+        customer_id time                                    
+        1           2014-01-01 00:45:00      (157.5, 283.46]
+                    2014-01-01 00:48:00      (31.288, 157.5]
+        2           2014-01-01 00:01:00      (157.5, 283.46]
+                    2014-01-01 00:04:00      (31.288, 157.5]
+
+        .. _custom-widths:
+
+        Using bins of `custom-widths`_:
+
+        >>> bins = [0, 200, 400]
+        >>> labels.bin(bins).head()
+                                        my_labeling_function
+        customer_id time                                    
+        1           2014-01-01 00:45:00           (200, 400]
+                    2014-01-01 00:48:00             (0, 200]
+        2           2014-01-01 00:01:00           (200, 400]
+                    2014-01-01 00:04:00             (0, 200]
+
+        .. _quantile-based:
+
+        Using `quantile-based`_ bins:
+
+        >>> labels.bin(4, quantiles=True).head() # (i.e. quartiles)
+                                                 my_labeling_function
+        customer_id time                                             
+        1           2014-01-01 00:45:00             (137.44, 241.062]
+                    2014-01-01 00:48:00              (43.848, 137.44]
+        2           2014-01-01 00:01:00             (241.062, 283.46]
+                    2014-01-01 00:04:00  (31.538999999999998, 43.848]
+
+        .. _labels:
+
+        Assigning `labels`_ to bins:
+
+        >>> labels.bin(2, labels=range(2)).head()
```

Comment:
maybe do `[low, medium, high]` as the example so it more clear what's going on

Reply:
Updated labels in [this](https://github.com/FeatureLabs/compose-ml/pull/18/commits/6d28dcb0a118097110d52216613d2422eeaba12a#diff-5d06a8d5b7b55549065b9c8f90f45edfR196) commit.